pkg/cmd/cli/backup/create.go

```diff
@@ -143,6 +143,7 @@ func (o *CreateOptions) Run(c *cobra.Command, f client.Factory) error {
 		return err
 	}
 
-	fmt.Printf("Backup %q created successfully.\n", backup.Name)
+	fmt.Printf("Backup request %q submitted successfully.\n", backup.Name)
+	fmt.Printf("Use `ark backup describe %s` for more details.\n", backup.Name)
```

Comment:
Nit: s/Use/Run
(same for restore)
reason: "use" is vague, "run" is more precise.

Reply:
Also, maybe this is my ignorance showing, but does `ark backup describe ...` provide additional information about the status of the backup request? If so, maybe let's ponder further ...

Reply:
What we have now is `ark backup get` and `ark backup describe`. `get` shows you the basic details in tabular format (the essential info being the phase - new/in progress/failed/completed). `describe` shows you slightly more information, but it's mostly the `spec` - what namespaces/resources/etc are included in the backup. Until we have #20 implemented, there's not really any info you can get other than seeing phase transitions.

Reply:
OK, good as is then.